src/main/scala/circt/stage/phases/MaybeSFC.scala

```diff
@@ -18,7 +19,16 @@ class MaybeSFC extends Phase {
 
   override def transform(annotations: AnnotationSeq) = {
     val sfcAnnotations: Option[AnnotationSeq] = view[CIRCTOptions](annotations).handover.get match {
-      case CIRCTHandover.CHIRRTL      => None
+      case CIRCTHandover.CHIRRTL      => if (annotations.exists {
+        // TODO[jiuyang]: This is a hotfix to ask SFC to resolve AOP.
+        //                Eventually, I think AOP should be moved to an early stage which out a custom firrl transform
```

Comment:
I'm not sure what this comment is saying. Do you mean "without a custom firrtl transform"?